cmd/agent/windows/service/service.go

```diff
@@ -57,6 +57,9 @@ loop:
 				log.Info("Received stop message from service control manager")
 				elog.Info(0x4000000c, config.ServiceName)
 				break loop
+			case svc.PreShutdown:
+				elog.Info(0x4000000d, config.ServiceName)
```

Comment:
Is there a reason we don't want this logged with `log.*` too like the other messages?

Reply:
I believed the logging was not initialized (but it is initialized) and I forgot to update the code.

Reply:
you reused the message from the message compiler, which means the event log will contain this message. 
May want to create a new specific message for the event log.